test/dummy/db/migrate/20210425231529_create_spina_admin_journal_journals.spina_admin_journal_engine.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+# This migration comes from spina_admin_journal_engine (originally 20201216152147)
+
+class CreateSpinaAdminJournalJournals < ActiveRecord::Migration[6.0] # :nodoc:
+  def change
+    create_table :spina_admin_journal_journals do |t|
+      t.string :name, null: false, default: 'Unnamed Journal'
```

Comment:
If your validations are working properly, it shouldn't be necessary to add a default unless you're explicitly skipping validations (and then you need to be careful for other reasons anyway).

Reply:
I believe I only ever did this because [Rubocop shouted at me](https://docs.rubocop.org/rubocop-rails/2.7/cops_rails.html#railsnotnullcolumn) one time for not having a default in a not-null column. I'm fully conscious that this default should never be used. I imagine the idea is to avoid db errors wherever possible.